lib/step.js

```diff
@@ -22,6 +22,7 @@ class Step {
   }
 
   run() {
+    this.startTime = new Date();
```

Comment:
If we place it in `steps.js` then we can't pass this value properly between states. Or I don't know how to do it. If you have any suggestion, it will be awesome